content/applications/sales/sales/ebay_connector/manage.rst

```diff
@@ -1,50 +1,102 @@
-======================
-How to list a product?
-======================
+=====================
+List products on eBay
+=====================
 
-Listing without variation
-=========================
+Listing (without product variants)
+==================================
 
-In order to list a product, you need to check the **use eBay** field on a product
-form. The eBay tab will be available.
+To list a product on eBay through Odoo, some necessary configurations must be completed on the
+desired product form.
 
-.. image:: manage/manage01.png
-  :align: center
+To access products, navigate to :menuselection:`Sales app --> Products --> Products`, and select the
+desired the product to be listed on eBay.
 
-When the **Use Stock Quantity** field is checked, the quantity sets on eBay will
-be the Odoo **Forecast Quantity**.
+Once on the product form, open the :guilabel:`eBay` tab. Under the :guilabel:`eBay` tab on the
+product form, check the box beside :guilabel:`Sell on eBay`.
```

Comment:
```suggestion
Once on the product form, open the :guilabel:`eBay` tab. Under the :guilabel:`eBay` tab, check the box beside :guilabel:`Sell on eBay`.
```
Just to avoid redundancy and keep things tight